### PR TITLE
Image

### PR DESCRIPTION
**Pull request by lannisterrr**
recording

### CODE REVIEW

File: components/image/image.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <div class="centered-row space-around-row component-border p-4">
+      <div class="image-container">
+        <img
+          src="https://picsum.photos/seed/picsum/200/200"
+          alt="responsive-image"
+          class="img-responsive"
+        />
+      </div>
+
+      <!-- round corner -->
```
Comment (saadahmadhere) [inline components/image/image.html]: If you're commenting here to specify the type of image, you should do the same for the previous div too.

**Review by saadahmadhere — COMMENTED** [review]
Good work overall.

File: components/image/image.html (hunk 1 above)
Comment (lannisterrr) [reply components/image/image.html]: ok I will do that.

**Review by ashishnnnnn — APPROVED** [review]
Code seems fine and preview also looks good

**Pull request merged**